tests_e2e/test_suites/agent_bvt.yml

```diff
@@ -1,8 +1,8 @@
 name: "AgentBvt"
 tests:
-  - "bvts/extension_operations.py"
```

Comment:
renamed directory to make it consistent with other tests: the directory is name is the same as the test suite's